__tests__/redux/actions/getArticles.test.js

```diff
@@ -85,16 +150,78 @@ describe('Should get profile', () => {
     store.clearActions();
   });
   test('should get user profile', async () => {
+    moxios.wait(() => {
+      const request = moxios.requests.mostRecent();
+      request.respondWith({
+        status: 200,
+        response: { message: 'dfedf' },
+      });
+    });
+
+    return store.dispatch(deleteArticle('deded')).then(() => {
+      expect(store.getActions().length).toEqual(1);
+    });
+  });
+});
+describe('Should Get My articles', () => {
+  beforeEach(() => {
+    moxios.install();
+  });
+  afterEach(() => {
+    moxios.uninstall();
+    store.clearActions();
+  });
+  it('should have PROFILE_INIT_START and PROFILE_INIT_SUCCESS', () => {
+    moxios.stubRequest('/articles/dcdcd', {
+      status: 200,
+      response: { data: {} },
+    });
+    store.dispatch(getMyArticles(1)).then(() => {
+      expect(store.getActions().length).toEqual(1);
+    });
+  });
+});
+describe('Should Upload image', () => {
+  beforeEach(() => {
+    moxios.install();
+  });
+  afterEach(() => {
+    moxios.uninstall();
+    store.clearActions();
+  });
+  test('should upload image', async () => {
+    const img = new File(['(^_^)'], 'smile.png', { type: 'image/png' });
     moxios.wait(() => {
       const request = moxios.requests.mostRecent();
       request.respondWith({
         status: 200,
         response: {
-          articles: {},
+          messahe: 'dfd',
         },
       });
     });
-    return store.dispatch(getUserProfile('ddffd')).then(() => {
+    return store.dispatch(uploadImage(img)).then(() => {
+      expect(store.getActions().length).toEqual(1);
+    });
+  });
+});
+describe('Should get profile', () => {
```

Comment:
@rukundoeric provide the name of the component under test in the describe ('........')